src/osd/OSD.cc

```diff
@@ -9015,6 +9015,9 @@ const char** OSD::get_tracked_conf_keys() const
     "clog_to_graylog_port",
     "host",
     "fsid",
+#ifdef HAVE_LIBFUSE
+    "osd_objectstore_fuse",
```

Comment:
it is already. probably you want to guard https://github.com/ceph/ceph/pull/9095/commits/ad3be40a66feb43e1a5295387b8dbcc22e30fdf9#diff-fa6c2eba8356ae1442d1bf749beacfdfR9015 with the `#ifdef` instead?